lib/child_process.js

```diff
@@ -7,13 +7,13 @@ const dgram = require('dgram');
 const assert = require('assert');
 const util = require('util');
 const debug = util.debuglog('child_process');
+const constants = require('constants');
```

Comment:
This is a change I was unsure about - before this, `constants` was lazy-loaded, but is now loaded via the module loader in the hopes that another package also required it (which would cache it). Is this better?